scripts/retry.js

```diff
@@ -24,22 +25,24 @@ class RetryOptions {
 }
 
 const parseInput = () => {
-  if (process.argv.length < 3) {
+  try {
+    const parsedResults = parseArgs({
+      allowPositionals: true,
+      options: {
+        maxAttempts: {
+          type: 'string',
```

Comment:
![image](https://github.com/aws-amplify/amplify-backend/assets/5849952/f04be9ab-b012-4c00-993b-8ca6573683a0)

Reply:
no number, we have to parse :(